pkgs/os-specific/darwin/apple-sdk-11.0/default.nix

```diff
@@ -54,5 +55,31 @@ let
     # questionable aliases
     configd = pkgs.darwin.apple_sdk.frameworks.SystemConfiguration;
     IOKit = pkgs.darwin.apple_sdk.frameworks.IOKit;
+
+    callPackage = newScope (lib.optionalAttrs stdenv.isDarwin rec {
+      inherit (pkgs.darwin.apple_sdk_11_0) stdenv;
+      darwin = pkgs.darwin.overrideScope (_: prev: { apple_sdk = prev.darwin.apple_sdk_11_0; });
```

Comment:
Does this mean the various `darwin.*`, like `Libsystem`, are still the ones built from source and therefore the 10.12 versions?

Reply:
It does. I pushed an updated commit that overrides everything in the `darwin` attribute that is based on the SDK version. That’s: `Libsystem`, `LibsystemCross`, `libcharset`, `libunwind`, `objc4`, `configd`, `IOKit`, `Security`, and `CF`.

Reply:
Thinking about it, there’s a possible gap with e.g., `binutils`. The version in the `stdenv` is based on the SDK, but I’m not sure how far I should go looking for packages that may possibly need to override something.

Reply:
I think this far is fine. If I get the Go packages building I will merge this.